rework_experiments/lib/navigation/wolt_modal_sheet_path.dart

```diff
@@ -0,0 +1,10 @@
+/// Class describing the path to navigate.
+class WoltModalSheetPath {
```

Comment:
I think Path is wrong term here. Path usually refers to a flow, a group of pages, route. Why don't we use similar naming as [`RouteSettings`](https://api.flutter.dev/flutter/widgets/RouteSettings-class.html) from the FlutterSDK?  The name can be `WoltModalSheetPageSettings`?

Reply:
Please also consider adding extensive documentation since this will be public API. You can copy paste from the Flutter framework.

Reply:
Despite the fact that they look similar, this class has a different semantic meaning. In current navigation implementation we don't just pass all pages to be shown, we register the possible pages and then choose what to show. Specifically this class describes an intent of showing one of the registered pages. And WoltModalSheetPathSettings is the registration description for a page. A page is registered once, but we can have 10 intents to show this page, and then we show 10 times this page.

For example:
/p1/p2/p1/p3/p1/p2

this address will show a navigation stack for our modal sheet like:
=== bottom of the stack===
page1
page2
page1
page3
page1
page2 => top of the stack that the user sees

It adds flexibility for navigation inside the modal sheet instead of just showing a bunch of pages one after another.